openmetadata-ui/src/main/resources/ui/src/components/GlobalSearchProvider/GlobalSearchSuggestions/GlobalSearchSuggestions.tsx

```diff
@@ -307,10 +313,17 @@ const GlobalSearchSuggestions = ({
         listHeight={220}
         notFoundContent={
           <div className="tw-flex tw-flex-col tw-w-full tw-h-56 tw-items-center tw-justify-center tw-pb-9">
-            <NoDataFoundSVG />
-            <Typography>No Results Found</Typography>
+            {isSuggestionsLoading ? (
+              <Loader size="small" />
+            ) : (
+              <>
+                <NoDataFoundSVG />
```

Comment:
Can we use antd `Empty` component here?
https://ant.design/components/empty/#header